python/ql/lib/semmle/python/Frameworks.qll

```diff
@@ -54,3 +54,6 @@ private import semmle.python.frameworks.Urllib3
 private import semmle.python.frameworks.Yaml
 private import semmle.python.frameworks.Yarl
 private import semmle.python.frameworks.Xmltodict
+private import semmle.python.frameworks.Cx_Oracle
+private import semmle.python.frameworks.Phoenixdb
+private import semmle.python.frameworks.Pyodbc
```

Comment:
If you could make these be sorted together with the rest of the list, that would be great :+1:

NIT: the missing newline, potentially you can add these settings to VS Code to never worry about that again ;)

```
    "files.trimTrailingWhitespace": true,
    "files.trimFinalNewlines": true,
    "files.insertFinalNewline": true,
```